### DISCUSSION TITLE
Pro VS Team

### DISCUSSION THREAD

**Discussion by GearStarter** [community/community#22971] [New to GitHub]
<p>Hey guys!</p>
<p>I have a small team: 5 developers.</p>
<p>And I want to use github repo for our project.</p>
<p>What tariff plan do we need?</p>
<p>Maybe can I to buy Pro plan for me or need Team plan for us?</p>

**Comment by seveas** [comment, marked as answer]
<p>For a 5-developer team, the team plan would be the right decision at $25/month</p>
